src/app/containers/Brand/index.jsx

```diff
@@ -1,22 +1,70 @@
 import React, { useContext } from 'react';
 import Brand from '@bbc/psammead-brand';
+import SkipLink from '@bbc/psammead-brand/skip-link';
+import ScriptLink from '@bbc/psammead-script-link';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import { bool } from 'prop-types';
+import { setPreferredVariantCookie } from '#contexts/UserContext/cookies';
 import { ServiceContext } from '#contexts/ServiceContext';
 
+const setCookie = (service, scriptLinkVariant) => {
+  return setPreferredVariantCookie(service, scriptLinkVariant);
+};
+
+const renderScriptLink = (
+  script,
+  service,
+  scriptLinkVariant,
+  url,
+  scriptLinkText,
+  offscreenText,
+) => (
+  <ScriptLink
+    script={script}
+    service={service}
+    href={url}
+    variant={scriptLinkVariant}
+    onClick={() => setCookie(service, scriptLinkVariant)}
+  >
+    <span aria-hidden>{scriptLinkText}</span>
```

Comment:
I think this needs to be `aria-hidden="true"`, as the aria tag expects a string not a boolean value!

Reply:
Added `"true"`